index.html

```diff
@@ -35,7 +35,7 @@
                         <a href="about.html" title="Over ons">Over Ons</a>
                     </div>
                     <div class="menu-item menu-button">
-                        <a href="nutrition.html" title="Voeding">Voeding</a>
+                        <a href="voeding.html" title="Voeding">Voeding</a>

```

Comment:
Sorry, mijn fout. Staat niet in het technisch ontwerp. Ze mogen Nederlands. Blijkt ook makkelijker achteraf.